gruntfile.js

```diff
@@ -92,13 +92,13 @@ module.exports = function(grunt) {
           'dist/popup.min.js': [
             pathLibs + 'jquery/dist/jquery.min.js',
             pathScripts + 'chrome-service.js',
-            pathScripts + 'usd-value-service.js',
+            pathScripts + 'currency-service.js',
             pathScripts + 'popup.js'
           ],
           'dist/notifier.min.js': [
```

Comment:
I think that makes sense, there might be some other stuff that we want to add that requires background processing and this might be the right place to put it if we rename this file.